data/data/openstack/masters/sg-master.tf

```diff
@@ -30,6 +42,19 @@ resource "openstack_networking_secgroup_rule_v2" "master_ingress_icmp" {
   description       = local.description
 }
 
+resource "openstack_networking_secgroup_rule_v2" "master_ingress_icmp_v6" {
+  count          = length(var.machine_v6_cidrs)
+  direction      = "ingress"
+  ethertype      = "IPv6"
+  protocol       = "icmp"
```

Comment:
I'm not sure it's `icmp` that we want:
https://opendev.org/openstack/neutron-lib/src/branch/master/neutron_lib/constants.py#L168

Reply:
https://docs.openstack.org/api-ref/network/v2/?expanded=create-security-group-rule-detail#create-security-group-rule

Reply:
done